build-config/resources/package/Ballerina.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+org = "ballerina"
+name = "wsdltool"
+version = "@toml.version@"
+authors = ["Ballerina"]
+keywords = ["wsdl"]
+license = ["Apache-2.0"]
+distribution = "2201.9.1"
```

Comment:
Do we have any dependency with the lang minor version ? If not we can update the distribution version to `2201.9.0`.